tests/testthat/test-metrics-quantile.R

```diff
@@ -371,9 +379,11 @@ test_that("wis is correct, 1 interval only - test corresponds to covidHubUtils",
   data_formatted <- merge(forecasts_formated, truth_formatted) %>%
     as_forecast()
 
-  eval <- suppressMessages(score(data_formatted,
-                                 count_median_twice = FALSE, metrics = metrics_no_cov_no_ae
-  ))
+  metrics <- metrics_quantile() %>%
+    select_metrics(select = c("wis"))
+  metrics$wis <- customise_metric(metrics$wis, count_median_twice = TRUE)
```

Comment:
I'm slightly confused as to why this works - the previous version had `count_median_twice = FALSE`, right?

Reply:
ok it seems like `count_median_twice = TRUE` and `FALSE` give the same result. This might be a bug, I'll investigate. 

But here I think this should be `count_median_twice = FALSE`. I suggested a change in the new PR to this PR.

Reply:
ok I think it's mostly just a stupid test - the forecast doesn't include the median, so changing the argument doesn't do anything...

Reply:
I added a new one to test that explicitly